### ISSUE TITLE
Handle Imgur albums

### ISSUE THREAD

**Issue by TheLonelyGhost** [TranscribersOfReddit/ToR_OCR#6]
## Scenario

> A post on /r/TranscribersOfReddit appears containing a link to an Imgur album.

## Current Result

OCR bot skips the album (it only transcribes if it is a direct link to an image)

## Expected Result

OCR bot pulls each image from the album and transcribes them

## Workaround

_None_

## Proposed fix

Have OCR bot detect if the link is of type `text/hml` on the imgur domain and, if so, scrape the page for each image, deferring to existing transcribing functionality from there.

**Comment by perryprog** [comment]
Check out the Imgur API, it’s what I was considering in ocr.space PR

**Comment by TheLonelyGhost** [comment]
Oh neat! Looks like the API doesn't require an API key if it's just accessing public data. Will definitely have to play with it to confirm

**Comment by perryprog** [comment]
Wait, you don't? I'm pretty sure it says you need authorization to do a GET on a album. I'll do some testing I suppose.

**Comment by perryprog** [comment]
Alright, I'll take this issue I suppose. Seems basic enough. I can start working on it for now, but it'll be helpful to get #10 closed.

**Comment by TheLonelyGhost** [comment]
Oops, I guess I misinterpreted [that part of the docs](https://apidocs.imgur.com/#authorization-and-oauth).

> NOTE: If your app is not only requesting public read-only information, then you may skip this step.

Looks like it actually meant one could skip the refresh token flow if working only with publicly accessible data, not that they could forgo API keys altogether.

**Comment by codingJWilliams** [comment]
Yeah you need an API key. I'll have a look and see if I can figure something out tonight. Imgur API keys are easy to get iirc.

**Comment by perryprog** [comment]
I already have an API key and image downloading, the only thing I had left to do was posting the comments.

**Comment by codingJWilliams** [comment]
Oh okay, look forward to seeing that then
